framework/src/modules/chain/chain.js

```diff
@@ -574,6 +614,20 @@ module.exports = class Chain {
 			}
 		});
 
+		this.blocks.on(EVENT_PRIORITY_CHAIN_DETECTED, ({ block }) => {
+			this.logger.info(
+				'Received EVENT_PRIORITY_CHAIN_DETECTED. Triggering synchronizer.'
+			);
+			this.synchronizer
```

Comment:
why not async await here?

Reply:
Not sure if use `await` or keep the promise resolved in detached mode as its a long process.

Reply:
ah i got ur point. 👍